app/src/App.css

```diff
@@ -97,7 +97,7 @@ body::-webkit-scrollbar-thumb {
   height: 100%;
   width: 200px;
 
-  background-color: floralwhite;
+  background-color: ghostwhite;
```

Comment:
Would it be helpful to use variables instead? So that way it's just once place you have to change for all of them. 
```css
:root {
  --bg-color: ghostwhite;
}
...
{
  ...
  background-color: var(--bg-color);
  ...
}
```

Reply:
I am so doing this.